src/App.tsx

```diff
@@ -1,8 +1,101 @@
-import React from 'react';
+import React, { useState, FC } from 'react';
+
+import { Button } from './components/Button';
+import { TodosList } from './components/TodosList';
+import { SortOptions } from './components/SortOptions';
+
+import { loadTodos } from './utils/todos';
+import { loadUsers } from './utils/users';
+import {
+  CompleteTodo,
+  Todo,
+  User,
+  SortOption,
+} from './constants/types';
+import { sortTodosByTitle, sortTodosByCompleteness, sortTodosByName } from './utils/helpers';
 import './App.css';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+export const App: FC = () => {
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [dataWasLoaded, setDataWasLoaded] = useState<boolean>(false);
```

Comment:
типы примитивов не указываются + make it easy
```suggestion
  const [isLoading, setLoading] = useState(false);
  const [isLoaded, setLoaded] = useState(false);
```